layers/+lang/emacs-lisp/extensions.el

```diff
@@ -15,17 +15,59 @@
 
 (defun emacs-lisp/init-emacs-builtin-emacs-lisp ()
 
-  (evil-leader/set-key-for-mode 'emacs-lisp-mode
-    "me$" 'lisp-state-eval-sexp-end-of-line
-    "meb" 'eval-buffer
-    "mec" 'spacemacs/eval-current-form
-    "mee" 'eval-last-sexp
-    "mer" 'eval-region
-    "mef" 'eval-defun
-    "mel" 'lisp-state-eval-sexp-end-of-line
-    "m,"  'lisp-state-toggle-lisp-state
-    "mtb" 'spacemacs/ert-run-tests-buffer
-    "mtq" 'ert)
+  (dolist (mode '(emacs-lisp-mode lisp-interaction-mode))
+    (evil-leader/set-key-for-mode mode
+      "me$" 'lisp-state-eval-sexp-end-of-line
+      "meb" 'eval-buffer
+      "mec" 'spacemacs/eval-current-form
+      "mee" 'eval-last-sexp
+      "mer" 'eval-region
+      "mef" 'eval-defun
+      "mel" 'lisp-state-eval-sexp-end-of-line
+      "m,"  'lisp-state-toggle-lisp-state
+      "mtb" 'spacemacs/ert-run-tests-buffer
+      "mtq" 'ert))
+
+  (if (configuration-layer/layer-usedp 'spacemacs)
```

Comment:
I don't quite understand. What if someone had Spacemacs-base and emacs-lisp
layers?
On Tue, Sep 22, 2015 at 9:45 PM Sylvain Benner <notifications@github.com>
wrote:

> In layers/+lang/emacs-lisp/extensions.el
> <https://github.com/syl20bnr/spacemacs/pull/3102#discussion_r40164641>:
>
> > -    "mtb" 'spacemacs/ert-run-tests-buffer
> > -    "mtq" 'ert)
> > +  (dolist (mode '(emacs-lisp-mode lisp-interaction-mode))
> > +    (evil-leader/set-key-for-mode mode
> > +      "me$" 'lisp-state-eval-sexp-end-of-line
> > +      "meb" 'eval-buffer
> > +      "mec" 'spacemacs/eval-current-form
> > +      "mee" 'eval-last-sexp
> > +      "mer" 'eval-region
> > +      "mef" 'eval-defun
> > +      "mel" 'lisp-state-eval-sexp-end-of-line
> > +      "m,"  'lisp-state-toggle-lisp-state
> > +      "mtb" 'spacemacs/ert-run-tests-buffer
> > +      "mtq" 'ert))
> > +
> > +  (if (configuration-layer/layer-usedp 'spacemacs)
>
> Do it the other way around, add emacs-builtin-emacs-lisp to the layer
> spacemacs and di your stuff in a post-init.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/syl20bnr/spacemacs/pull/3102/files#r40164641>.
>